test cases/common/271 env in generator.process/generate_main.py

```diff
@@ -7,5 +7,5 @@
 
 with open(sys.argv[1], 'r') as infile, \
      open(sys.argv[2], 'w') as outfile:
-    
+
```

Comment:
While I broadly agree with the idea of this change, it's not exactly an "optimization"...

Reply:
Reverting whitespace changes.